msldap/examples/msldapclient.py

```diff
@@ -184,7 +186,7 @@ async def do_query(self, query, attributes = None):
 			async for entry, err in self.connection.pagedsearch(query, attributes):
 				if err is not None:
 					raise err
-				print(entry)
```

Comment:
I know the current version is not the best, but using JSON will break a lot of things because the `enrty` variable can hold objects which are not JSON serializable.

Reply:
I canceled right JSON output